avocado/core/__init__.py

```diff
@@ -166,6 +166,16 @@ def register_core_options():
                          default='utf-8',
                          help_msg=help_msg)
 
+    # All settings starting with 'runner.' will be passed to runner
+    # exec-test runner config
+    help_msg = ('Use a custom exit code list to consider a test as skipped. '
+                'This is only used by exec-test runners. Default is [130].')
```

Comment:
Won't this `130` exit code bring surprise to users, both old (specially) and new?  I'd really let users opt in and choose the special meaning of their exit codes.

Reply:
The idea was: if some script is using the right exit code when CTRL+C (130), then the intention was to skip the test. But we can have an empty list as default if you think that is better.

Reply:
I looked that exit status code up, and guessed that was your idea. But, TBH, among the surprise factor, I don't think a program *interrupted* means it was skipped... that's my main problem with this proposal.

Reply:
I have the impression that could be. But ok, no worries I will make `default=[]` on a v2.